landingcorp/index.html

```diff
@@ -9,9 +9,15 @@
     <link rel="manifest" href="manifest.webmanifest" />
     <link rel="apple-touch-icon" href="icon.png" />
     <!-- Place favicon.ico in the root directory -->
-
+    <title>LandingCrop</title>
+    <link rel="stylesheet" href="./css/bootstrap.min.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800&display=swap"
+      rel="stylesheet"
+    />
     <link rel="stylesheet" href="css/normalize.css" />
     <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="./css/style.css" />
```

Comment:
Since Bootstrap 4 packs normalization rules, we can 
1. remove css/normalize.css and
2. use main.scss to define our template-specific styles instead of the _template.scss file (which we can actually remove) :wink:

In the current setup, the main.scss file is removed, but we are still referencing it. Furthermore we reference a style.css of which I don't see a source file so there are some broken references that need to be taken care of.

I would propose something along the following diff:

```diff
-    <link rel="stylesheet" href="css/normalize.css" />
     <link rel="stylesheet" href="css/main.css" />
-    <link rel="stylesheet" href="./css/style.css" />
```